internal/common/validations.go

```diff
@@ -5,13 +5,39 @@ package common
 
 import (
 	"github.com/go-openapi/swag"
+	"github.com/openshift/assisted-service/models"
 )
 
 // IsAgentCompatible checks if the given agent image is compatible with what the service expects.
 func IsAgentCompatible(expectedImage, agentImage string) bool {
 	return agentImage == expectedImage
 }
 
+var ClusterValidationIDs [] string = [] string {
```

Comment:
Due to this duplication (with IDs listed in models) it is likely that if we ever add/remove a validation this list will be outdated

Reply:
I don't feel that these are duplications. Show me where there is an easily accessible list of validations by type and I will agree with you.

I get that there may be extra work involved if/when adding a new validation as the ID will need to be present here in order for the validation to be "ignorable"

Reply:
Maybe adding something to the code generator to make these lists would be a good idea?

Reply:
Look at models/cluster_validation_id.go
You can call:
validation := NewClusterValidationID(<string>)
err := validation.Validate(nil)

See [here](https://github.com/openshift/assisted-service/blob/5f07924ba6389f0353f9b9023cf5ce9f2f3cc63b/internal/host/hostcommands/instruction_manager.go#L268) for an example